lib/s3middleware/tagging.js

```diff
@@ -3,7 +3,8 @@ const { parseString } = require('xml2js');
 const errors = require('../errors');
 const escapeForXml = require('./escapeForXml');
 
-const tagRegex = new RegExp(/[^a-zA-Z0-9 +-=._:/]/g);
+const tagRegex =
+new RegExp(/[^a-zA-Z0-9\u0032\u0043\u0045-\u0058\u0061\u0064-\uFFFF +-=._:/]/g);
```

Comment:
Can you add a comment explaining the range of characters accepted? Also, please move the regex to `/lib/constants.js`.